apps/cli/src/commands/doctor.ts

```diff
@@ -11,102 +11,104 @@ export default class DoctorCommand extends Command {
 
-                // Check Docker version
-                if (!DoctorCommand.isDockerVersionValid(dockerVersion)) {
-                    throw new Error(
-                        `Unsupported Docker version. Minimum required version is ${DoctorCommand.MINIMUM_DOCKER_VERSION}. Installed version is ${dockerVersion}.`,
-                    );
-                }
-            } catch (error: unknown) {
-                if (error instanceof Error) {
-                    this.error(
-                        "Docker is required but not installed or the command execution failed. Please refer to the Docker documentation for installation instructions: https://docs.docker.com/get-docker/",
-                    );
-                }
+            const v = semver.coerce(dockerComposeVersion);
+            if (
+                v !== null &&
+                !semver.gte(v, DoctorCommand.MINIMUM_DOCKER_COMPOSE_VERSION)
+            ) {
+                throw new Error(
+                    `Unsupported Docker Compose version. Minimum required version is ${DoctorCommand.MINIMUM_DOCKER_COMPOSE_VERSION}. Installed version is ${v}.`,
+                );
             }
-
-            try {
-                const dockerComposeVersion =
-                    await DoctorCommand.checkDockerComposeVersion();
-
-                // Check if the Docker Compose version is valid
-                if (
-                    !DoctorCommand.isDockerComposeVersionValid(
-                        dockerComposeVersion,
-                    )
-                ) {
-                    throw new Error(
-                        `Unsupported Docker Compose version. Minimum required version is ${DoctorCommand.MINIMUM_DOCKER_COMPOSE_VERSION}. Installed version is ${dockerComposeVersion}.`,
-                    );
-                }
-            } catch (error: unknown) {
-                if (error instanceof Error) {
-                    this.error(
-                        "Docker Compose is required but not installed or the command execution failed. Please refer to the Docker Compose documentation for installation instructions: https://docs.docker.com/compose/install/",
-                    );
-                }
+        } catch (e: unknown) {
+            if ((e as any).exitCode === 125) {
+                throw new Error(
+                    "Docker Compose is required but not installed or the command execution failed. Please refer to the Docker Compose documentation for installation instructions: https://docs.docker.com/compose/install/",
+                );
+            } else {
+                throw e;
             }
+        }
 
-            // Check Docker Buildx version and riscv64 support
+        return true;
+    }
+
+    private async checkBuildx(): Promise<true | never> {
+        try {
             const { stdout: buildxOutput } = await execa("docker", [
                 "buildx",
                 "ls",
+                "--format",
+                "{{.Platforms}}",
```

Comment:
I have already installed the `buildx` but `doctor` generates this error : 

```
    Error: Docker Buildx is required but not installed. Please refer to the Docker Desktop documentation for installation instructions: https://docs.docker.com/desktop/

```

```shell
docker buildx ls
```

Output: 

```
NAME/NODE       DRIVER/ENDPOINT STATUS  BUILDKIT PLATFORMS
default         docker                           
  default       default         running v0.13.1  linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/arm64, linux/riscv64, linux/ppc64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
desktop-linux * docker                           
  desktop-linux desktop-linux   running v0.12.5  linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6

```

Reply:
`docker  buildx ls --format {{.Platforms}}` output: 

```
unknown flag: --format
See 'docker buildx ls --help'.
```

My version : 
```github.com/docker/buildx v0.12.1-desktop.4 6996841df2f61988c2794d84d33205368f96c317```

Reply:
That's why it's getting into the `exitCode === 125` branch.

If using `--format`, we may need to add a check for a minimum build version.

My version is this:

```
❯ docker buildx version
github.com/docker/buildx v0.14.0 171fcbe
```

Reply:
Since the old version does not effect on `sunodo`, I would suggest to remove the `--format` from `buildx` check.

Reply:
Acctually, I think using at least 0.13.0 would be great for better stability in processing the output.

Reply:
Sent a fixup to check the buildx version .